testing/xtrabackup/APKBUILD

```diff
@@ -0,0 +1,80 @@
+# Contributor: Bartłomiej Piotrowski <bpiotrowski@alpinelinux.org>
+# Contributor: Andriy Kushnir <me@orhideous.name>
+# Maintainer: Andriy Kushnir <me@orhideous.name>
+
+pkgname=xtrabackup
+pkgver=2.4.5
+pkgrel=0
+pkgdesc='MySQL non-blocking backup solution for InnoDB and XtraDB data'
+arch=all
+url='http://www.percona.com/software/percona-xtrabackup/'
+license='GPL'
+depends='
+  libaio
+  libcurl
+  libev
+  libgcc
+  libgcrypt
+  libstdc++
+  musl
+  zlib
+  perl
+  perl-dbd-mysql
+'
+makedepends='
+  bash
+  bison
+  cmake
+  flex
+  vim
+  ncurses-dev
+  curl-dev
+  libev-dev
+  libaio-dev
+  libgcrypt-dev
+  zlib-dev
+'
+source="
+  https://www.percona.com/downloads/XtraBackup/Percona-XtraBackup-$pkgver/source/tarball/percona-xtrabackup-$pkgver.tar.gz
+  sigevent_musl.patch
+"
+
+prepare() {
```

Comment:
Prepare that applies patches only is no longer needed, the 'default_prepare' does that. This can be removed. You just need to set `builddir="$srcdir"/percona-xtrabackup-$pkgver`.